docs/en/cloud/AWS-EMR.md

```diff
@@ -31,93 +31,192 @@ Blob Store.
 ```console
 $ aws emr create-default-roles
 ```
-2. The Alluxio bootstrap script is hosted in a publicly readable
-[S3 bucket](https://alluxio-public.s3.amazonaws.com/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh).
-This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh`
-The bootstrap script only requires a root UFS URI as an argument.
-Additional options can be seen in the comments at the top of the bootstrap script.
+
+  {% endcollapsible %}
+{% endaccordion %}
+
+{% accordion setup create cluster %}
+  {% collapsible Run the aws create-cluster command with the bootstrap action %}
+The `create-cluster` command requires passing in multiple flags to successfully execute:
+- `release-label`: The version of EMR to install with.
+The current version of Alluxio is compatible with `emr-5.25.0`.
+- `custom-ami-id`: The Alluxio Marketplace Enterprise Edition AMI ID.
+To find the latest AMI ID for a particular region, run the following AWS CLI command:
+```console
+aws ec2 describe-images \
+--owners 'aws-marketplace' \
+--filters 'Name=product-code,Values=6axlevynmkcnoq7ximzar80bx' \
+--query 'sort_by(Images, &CreationDate)[-1].[ImageId]' \
+--output text \
+--region us-east-1
+```
+where the value for `--region` can be set to the desired region.
+
+- `instance-count`: The number of nodes to provision for the cluster.
+- `instance-type`: The instance type to provision with.
+Make sure you pick an instance type supported by the Alluxio marketplace AMI.
+Note that your account can be limited in the number of instances you can launch; check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Limits:).
+The default instance type for the AMI is `r4.4xlarge`.
+- `applications`: Specify `Name=Spark Name=Presto Name=Hive` to bootstrap the three additional services
+- `name`: The EMR cluster name
+- `bootstrap-actions`:
+  - `Path`: The path to the bootstrap script, hosted in a publicly readable S3 bucket: `s3://alluxio-public/emr/{{site.ALLUXIO_RELEASED_VERSION}}/alluxio-emr.sh`
+  - `Args`: The arguments passed to the bootstrap script.
+  The first argument, the root UFS URI, is required.
+  This S3 URI designates the root mount of the Alluxio file system and should be of the form `s3://bucket-name/mount-point`.
+  The mount point should be a folder; follow [these instructions](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-folder.html) to create a folder in S3.
+  You can also specify additional Alluxio properties as a delimited list of key-value pairs in the format `key=value`.
+  For example, `alluxio.user.file.writetype.default=CACHE_THROUGH` instructs ALluxio to write files synchronously to the underlying storage system.
+  See more about [write type options](https://docs.alluxio.io/ee/user/stable/en/Architecture-DataFlow.html?q=write%20types#data-flow-write).
```

Comment:
do we really want the query string in this URL? I don't think we should be highlighting terms on the page

Reply:
the highlighting is okay to me but it breaks the auto-nav to the data flow: write section...